p2p-specs/p2p-interface.md

```diff
@@ -494,7 +499,29 @@ The response MUST be encoded as an SSZ-container.
 
 The response MUST consist of a single `response_chunk`.
 
-##### `PooledUserOpsByHashes`
+#### PooledUserOpHashes
+
+**Protocol ID:** `/account_abstraction/erc4337/req/pooled_user_op_hashes/1/`
+
+Request Content:
+
+```
+(
+  mempool: Bytes32
+)
+```
+
+Response Content: 
+```
+(
+  List[Bytes32, MAX_OPS_PER_REQUEST]
```

Comment:
How does sending multiple `PooledUserOpsByHashes` works? there is no state so sending the same request will get the same response - the "first" (by whatever ordering defined by the receiving bundler) userops.

**Suggestion**
The PooledUserOpsByHashes request MAY have "offset" parameter.
the receiving bundler should skip the first "offset" hashes, and send the next ones.
Actual ordering of userops is not important, as long as it is deterministic within the receiving bundler (e.g. sort by hash or sort by receiving time)

Since time elapses between the first and second request, some userops might be duplicated (or skipped) between the two requests, but that is still ok.

Reply:
> How does sending multiple `PooledUserOpsByHashes` works? there is no state so sending the same request will get the same response - the "first" (by whatever ordering defined by the receiving bundler) userops.
> 
This message type was expected to be used by bundlers during startup to request a list of userops in the mempool from its connected peers.

Reply:
Did this clarify your question @drortirosh?

Reply:
This has now been addressed @drortirosh